pkg/operation/botanist/controlplane.go

```diff
@@ -115,7 +115,8 @@ func (b *Botanist) deleteNamespace(name string) error {
 // the cloud-controller-manager and the kube-controller-manager
 func (b *Botanist) DeployCloudMetadataServiceNetworkPolicy() error {
 	values := map[string]interface{}{
-		"cloudProvider": b.Seed.CloudProvider,
+		"cloudProvider":  b.Seed.CloudProvider,
+		"loggingEnabled": controllermanagerfeatures.FeatureGate.Enabled(features.Logging),
```

Comment:
With the current network policies it is impossible, because the `cloud-metadata-service` policies are enabling egress traffic for all pods, except some control plane pods like `kube-apiserver`, `kube-controller-manager`, etc.
This is the only place where we could introduce additional exception, which will disable the egress traffic from the kibana pod, otherwise it could talk with everything except cloud metadata service.

Reply:
OK. Then we either don't add this network policy here right now (@mvladev is anyway reworking everything with #873), or we change the cloud-metadata-service network policy so that the logging policy can be separate from this (high risk).

Reply:
(Background: We are progressing with the control plane extensibility, and with it this cloud-metadata-service Helm chart will go away. Any dependency to other components (like logging now) will complicate this.)

Reply:
@mvladev is working on #873 which will allow us to remove the logging specifics from the `cloud-metadata-service` chart.

Reply:
With the current setup of the network policies, every new network policy will depend on the `cloud-metadata-service`, because it white-lists every app which has no label `app=kubernetes`.
So this is only a temporary decision until Martin Vladev gets his PR merged.